internal/status/status.go

```diff
@@ -0,0 +1,26 @@
+package status
+
+import (
+	"context"
+
+	dockerparser "github.com/novln/docker-parser"
+
+	"github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
+)
+
+func Refresh(ctx context.Context, k StatusClient, tempo v1alpha1.Microservices, status *v1alpha1.MicroservicesStatus) (bool, error) {
+	tempoImage, err := dockerparser.Parse(tempo.Spec.Images.Tempo)
```

Comment:
I know this comes from a previous PR and you are just refactoring but: should we show this information in the query too?

Reply:
for the query image? sure

Reply:
Done.